Microsoft.RestApi.RestSplitter/RestSplitter.cs

```diff
@@ -275,17 +276,17 @@ private static void WriteToc(string sourceRootDir, string targetRootDir, OrgsMap
             }
         }
 
-        private static SortedDictionary<string, List<SwaggerToc>> SplitSwaggers(string sourceRootDir, string targetApiVersionDir, ServiceInfo service, MappingConfig mappingConfig, RepoFile repoFile, string version)
+        private static SortedDictionary<string, List<SwaggerToc>> SplitSwaggers(string sourceRootDir, string targetRootDir, string targetApiVersionDir, ServiceInfo service, MappingConfig mappingConfig, RepoFile repoFile, string version)
         {
             var subTocDict = new SortedDictionary<string, List<SwaggerToc>>();
-
+            var sourceSwaggerMappingDict = new Dictionary<string, string>();
             foreach (var swagger in service.SwaggerInfo)
             {
                 var subGroupName = swagger.SubGroupTocTitle ?? string.Empty;
                 var targetDir = FileUtility.CreateDirectoryIfNotExist(Path.Combine(targetApiVersionDir, service.UrlGroup, subGroupName.TrimSubGroupName()));
                 var sourceFile = Path.Combine(sourceRootDir, swagger.Source.TrimEnd());
 
-                var restFileInfo = RestSplitHelper.Split(targetDir, sourceFile, swagger.Source.TrimEnd(), mappingConfig.UseServiceUrlGroup ? service.UrlGroup : service.TocTitle, service.TocTitle, swagger.OperationGroupMapping, mappingConfig, repoFile);
+                var restFileInfo = RestSplitHelper.Split(targetDir, sourceFile, swagger.Source.TrimEnd(), mappingConfig.UseServiceUrlGroup ? service.UrlGroup : service.TocTitle, service.TocTitle, swagger.OperationGroupMapping, mappingConfig, repoFile, out string sourceSwaggerUrl);
```

Comment:
maybe I am wrong...
why do we pass the sourceSwaggerUrl into complex Split method
can we have another method to handle?